packages/theme-sections/theme-sections.js

```diff
@@ -212,13 +212,16 @@ function normalizeContainers(containers) {
 }
 
 if (window.Shopify.designMode) {
-  document.addEventListener('shopify:section:load', function(event) {
-    var id = event.detail.sectionId;
-    var container = event.target.querySelector(
-      '[' + SECTION_ID_ATTR + '="' + id + '}"]'
+  document.addEventListener('shopify:section:load', event => {
+    const container = event.target;
+    const containerDataset = container.dataset;
+
+    // Hack solution - pulling the first data-attribute since the identifier is unknown
+    const sectionData = JSON.parse(
+      containerDataset[Object.keys(containerDataset)[0]]
```

Comment:
Currently this code makes the assumption that this specific data-attribute will always be returned first. There has to be a better way to target the attribute to get the type value. Any ideas?